<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Developer Mode</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="step0_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Imput Id, Date, 3 pins&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>step 0: setup</string>
    </property>
   </widget>
   <widget class="QPushButton" name="step1_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;set speed and time&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>step 1: metering</string>
    </property>
   </widget>
   <widget class="QPushButton" name="step2_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>step 2: incubation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="step3_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>step 3: lysis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="step4_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>step 4: netrualization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="step5_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>step 5: hybridization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="step6_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>step 6: washing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="step7_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>step 7: detection</string>
    </property>
   </widget>
   <widget class="QToolButton" name="step8_dev">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string>step 8: Go</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="spspin_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>simple spin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spled_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>simple led</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="spvalving_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>simple valving</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="n_dev">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>n for channels</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_dev">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x for angular position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rpm_dev">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>rpm for speed</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_dev">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>t for time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="a0_dev">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>a0, a1, f for oscillation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="a1_dev">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="f_dev">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="r_dev">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>260</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>r for linear position</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sppositioning_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>simple positioning</string>
    </property>
   </widget>
   <widget class="QComboBox" name="imgprocess_dev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Subtract</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Findcircle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Patient ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pid_dev">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>110</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="motorstop_dev">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop motor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Exposure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="expo_dev">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>350</y>
      <width>110</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <zorder>step0_dev</zorder>
   <zorder>step1_dev</zorder>
   <zorder>step2_dev</zorder>
   <zorder>step3_dev</zorder>
   <zorder>step4_dev</zorder>
   <zorder>step5_dev</zorder>
   <zorder>step6_dev</zorder>
   <zorder>step7_dev</zorder>
   <zorder>step8_dev</zorder>
   <zorder>spspin_dev</zorder>
   <zorder>spled_dev</zorder>
   <zorder>spvalving_dev</zorder>
   <zorder>n_dev</zorder>
   <zorder>label</zorder>
   <zorder>x_dev</zorder>
   <zorder>label_2</zorder>
   <zorder>rpm_dev</zorder>
   <zorder>label_3</zorder>
   <zorder>t_dev</zorder>
   <zorder>label_4</zorder>
   <zorder>a0_dev</zorder>
   <zorder>label_5</zorder>
   <zorder>a1_dev</zorder>
   <zorder>f_dev</zorder>
   <zorder>r_dev</zorder>
   <zorder>label_6</zorder>
   <zorder>imgprocess_dev</zorder>
   <zorder>label_11</zorder>
   <zorder>pid_dev</zorder>
   <zorder>label_12</zorder>
   <zorder>dateEdit</zorder>
   <zorder>motorstop_dev</zorder>
   <zorder>sppositioning_dev</zorder>
   <zorder>label_7</zorder>
   <zorder>expo_dev</zorder>
   <zorder>exit</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../images/MDX.qrc"/>
 </resources>
 <connections/>
</ui>
